music/src/core/data.ts

```diff
@@ -136,8 +143,8 @@ export abstract class DataConverter {
   readonly numSegments: number;              // Number of steps for conductor.
   abstract readonly depth: number;           // Size of final output dimension.
   abstract readonly endTensor: tf.Tensor1D;  // Tensor marking segment end.
-  abstract readonly NUM_SPLITS: number;  // Const number of conductor splits.
-  abstract readonly SEGMENTED_BY_TRACK: boolean;  // Segments are tracks.
+  readonly NUM_SPLITS: number = 0;  // Const number of conductor splits.
```

Comment:
if these aren't abstract, should they be moved below with their brethren NOTE_OFF, etc?

Reply:
no, because they are needed in all subclasses